### DISCUSSION TITLE
recipes/desserts/cake/chez-panisse-almond-torte/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#983] [Comments]
# recipes/desserts/cake/chez-panisse-almond-torte/

A collection of my recipes.

https://nicholaswilde.io/recipes/desserts/cake/chez-panisse-almond-torte/

**Comment by nicholaswilde** [comment]
Move the buttering of the pan to the top.
Add vanilla to eggs
Add parchment to bottom of pan.

**Reply by nicholaswilde** [reply]
Baked for 1 hour
